### PR TITLE
Update the documentation making it more readable

### PR DESCRIPTION
**Pull request by vnazar**
- Improve summary
- Add Fintoc logo
- Add ToC
- Improve instructions (Get your keys and setup, Run, Run + Docker)
- Remove unnecessary print examples of `make` (docker-compose)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,86 @@
...
-</td>
-</tr>
-</table>
-
-And enter it with:
+Clone the repository using your prefered terminal using any of the following commands:
+
+| Protocol   | Command                                                        |
+|------------|----------------------------------------------------------------|
+| HTTPS      | ```$ git clone https://github.com/fintoc-com/sample-app.git``` |
```
Comment (lezorich) [inline README.md:42]: Here the url should be quickstart right?

File: README.md [hunk 2]
```diff
@@ -1,77 +1,86 @@
...
+| SSH        | ```$ git clone git@github.com:fintoc-com/quickstart.git```     |
+| GitHub CLI | ```$ gh repo clone fintoc-com/sample-app```                    |
```
Comment (lezorich) [inline README.md:44]: Here too. Is the old repo name

**Review by lezorich — APPROVED** [review]
LGTM! I left two comments only but the rest looks great 👌

File: README.md (hunk 2 above)
Comment (vnazar) [reply README.md:44]: You're right, i fix it now

File: README.md (hunk 1 above)
Comment (vnazar) [reply README.md:42]: You're right, i fix it now

**Review by jpjofreb — APPROVED** [review]
LGTM

**Pull request merged**